components/02-elements/button/_button.scss

```diff
@@ -1,121 +1,113 @@
-$button-font-family               : $font-family-sans-serif !default;
-$button-font-size                 : $font-size-base !default;
-$button-font-color                : $white !default;
+$button__height           : 48px !default;
+$button__width            : 48px !default;
+$button__padding          : 0 $padding-base !default;
+$button__border           : none !default;
+$button__border-radius    : 100px !default;
+$button__border-hover     : none !default;
+$button__background       : $color-primary !default;
+$button__background-hover : $gray-dark !default;
+$button__outline          : none !default;
+$button__font-family      : $font-family-base-bold !default;
+$button__font-size        : $font-size-base !default;
+$button__font-color       : $white !default;
+$button__font-color-hover : $color-primary !default;
 
 // Secondary
-$button-secondary-border          : 3px solid $gray-light !default;
-$button-secondary-hover-border    : $color-primary !default;
-$button-secondary-background      : $white !default;
-$button-secondary-font-color      : $gray-light !default;
-$button-secondary-hover-font-color: $color-primary !default;
+$button__border--secondary           : 1px solid $color-primary !default;
+$button__border-hover--secondary     : $color-primary !default;
+$button__background--secondary       : $white !default;
+$button__background-hover--secondary : $color-primary !default;
+$button__font-color--secondary       : $color-primary !default;
+$button__font-color-hover--secondary : $gray-dark !default;
 
 // Icon
-$button-icon-padding              : 0 !default;
-$button-icon-size                 : 20px !default;
-$button-icon-content-margin       : 0 15px 0 0 !default;
-$button-icon-border-width         : 3px !default;
-$button-icon-border               : $button-icon-border-width solid $gray-light !default;
-$button-icon-hover-border         : $color-primary !default;
-$button-icon-fill                 : $gray-light !default;
-$button-icon-hover-fill           : $color-primary !default;
-$button-icon-background           : $white !default;
-$button-icon-color                : $gray-light !default;
-$button-icon-hover-color          : $color-primary !default;
+$button-icon__padding          : 0 !default;
+$button-icon__size             : 20px !default;
+$button-icon__content-margin   : 0 $margin-base 0 0 !default;
+$button-icon__border-width     : 3px !default;
+$button-icon__border           : $button-icon__border-width solid $color-primary !default;
+$button-icon__border-hover     : $gray-dark !default;
+$button-icon__fill             : $white !default;
+$button-icon__fill-hover       : $color-primary !default;
+$button-icon__background       : $color-primary !default;
+$button-icon__background-hover : $gray-dark !default;
+$button-icon__color            : $gray-light !default;
+$button-icon__color-hover      : $color-primary !default;
 
 //fluid
 $button-fluid-width               : 100% !default;
```

Comment:
and this variable is align to which element? ;)

Reply:
line 110:
![image](https://user-images.githubusercontent.com/12099511/32044500-063740b4-ba3e-11e7-9bb6-f082b6f04d2b.png)

Reply:
But i guess its not BEM so I renamed it to `button__width--fluid`

Reply:
👍